packages/auto-chart/src/AutoChart.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+interface Props {
+  id: string;
+  className?: string;
+  width?: number;
+  height?: number;
+  data?: any[];
+  toolbar?:boolean; 
+  development?: boolean;
+  title?: string;
+  description?: string;
+  purpose?: 'Comparison' | 'Trend' | 'Distribution' | 'Rank' | 'Proportion' | 'Composition';
+};
```

Comment:
props like `toolbar` , `development` are not good named. we shall re-design these APIs.

leave a TODO tag here, or just remove props for now.

Reply:
already delete